external/@wordpress/blocks/index.js

```diff
@@ -19,14 +28,20 @@ export function registerBlock( namespace, block, settings ) {
  * @return {?Object}           Block settings
  */
 export function getBlockSettings( namespace, block ) {
+	if ( ! blocks[ namespace ] || ! blocks[ namespace ][ block ] ) {
+		return null;
+	}
 
+	return blocks[ namespace ][ block ];
 }
 
 /**
  * Returns all registered blocks.
  *
- * @return {Object} Block settings keyed by block name
+ * @return {Array} Blocks settings
```

Comment:
> Regarding the name, I'm leaning towards removing the namespace and encouraging a convention

_Encouraging_ a convention is not enough, it needs to be _enforced_ otherwise people won't follow it.

Though this can be done either way -

```js
if ( ! /^\w+/\w+$/.test( blockName ) ) {
    throw new Error( 'Use a namespace please' );
}
```